src/dotnet/Core/Constants.DebugMode.cs

```diff
@@ -58,7 +58,7 @@ public static bool AudioPlaybackPlayMyOwnAudio
         public static readonly bool TranscriberGoogle = false;
 
         // Search
-        public static readonly bool OpenSearchRequest = false;
+        public static readonly bool OpenSearchRequest = true;
 
```

Comment:
Do you enable debug mode for OpenSearch on purpose?

Reply:
fixed, thanks